tests/typing_example.py

```diff
@@ -167,7 +167,7 @@ class Validated:
             attr.validators.instance_of(C), attr.validators.instance_of(D)
         ),
     )
-    e: str = attr.ib(validator=attr.validators.matches_re(r"foo"))
+    e: str = attr.ib(validator=attr.validators.matches_re(re.compile(r"foo")))
```

Comment:
I think we should keep both, but I'll do it myself to save ourselves another review round.

Reply:
ah nevermind, `f` has a str :)

Reply:
yes, couldn't remember the alphabet and i knew you would complain about `e2` 😛